cddl/usr.sbin/dtrace/tests/common/Makefile

```diff
@@ -4,6 +4,12 @@
 # FreeBSD, and docsExamples doesn't seem amenable to automated testing.
 
 TESTSDIR=	${TESTSBASE}/cddl/usr.sbin/dtrace/common
+
+
+# A few tests how did not compile in CHERI have been removed from here.
+# The problems were missing dependencies.
+# Tests removed: json usdt
+
```

Comment:
Maybe move them to ``TESTS_SUBDIRS_NOCHERI` to keep building them for other targets?